build-support/scripts/protobuf.sh

```diff
@@ -123,7 +128,7 @@ function postprocess_protobuf_code {
 function generate_mog_code {
     local mog_order
 
-    mog_order="$(go list -tags "${GOTAGS}" -deps ./proto/pb... | grep "consul/proto")"
+    mog_order="$(go list -tags "${GOTAGS}" -deps ./proto/pb... | grep "consul/proto/")"
```

Comment:
is adding a `/` here intensional?

Reply:
Yep! It causes us to only match `proto` and not `proto-public`.